universal-application-tool-0.0.1/project/plugins.sbt

```diff
@@ -2,3 +2,4 @@
 addSbtPlugin("com.typesafe.play" % "sbt-plugin" % "2.8.8")
 addSbtPlugin("com.typesafe.sbt" % "sbt-play-ebean" % "6.0.0")
 addSbtPlugin("name.de-vries" % "sbt-typescript" % "2.6.2")
+addSbtPlugin("net.virtual-void" % "sbt-dependency-graph" % "0.10.0-RC1")
```

Comment:
thought (non-blocking): would generally prefer this kind of change be done in its own PR

Reply:
didn't mean to commit this file! will revert the change.